bin/test-merges.js

```diff
@@ -0,0 +1,296 @@
+#! /usr/bin/env node
+"use strict";
+
+/* eslint-disable no-console */
+/* this is a node script; primary interaction is via console */
+
+const Task = require("co-task");
+const process = require("process");
+const path = require("path");
+const GitHubApi = require("github");
+const shelljs = require("shelljs");
+
+const github = new GitHubApi({debug: false});
+
+// some of our API requests need to be authenticated
+let token = process.env.AS_PINE_TOKEN;
+github.authenticate({type: "token", token});
+
+// note that this token MUST have the public_repo scope set in the github API
+
+const AS_REPO_OWNER = process.env.AS_REPO_OWNER || "mozilla";
+const AS_REPO_NAME = process.env.AS_REPO_NAME || "activity-stream";
+const AS_REPO = `${AS_REPO_OWNER}/${AS_REPO_NAME}`;
+const OLDEST_PR_DATE = "2017-02-13";
+const HG = "hg"; // mercurial
+const HG_BRANCH_NAME = "pine";
+const ALREADY_PUSHED_LABEL = "pushed-to-pine";
+const TREEHERDER_PREFIX = "https://treeherder.mozilla.org/#/jobs?repo=pine&revision=";
+
+// Path to the directory where the export/commit operations will be
```

Comment:
nit: probably good to refer to this as "Path to the *working* directory"… given how big this comment block is and reinforcing that it's all to describe `WORKING_DIR`